Gems/AWSCore/Code/Tools/ResourceMappingTool/README.md

```diff
@@ -89,3 +89,13 @@ you can create the virtualenv manually.
    ```
    $ python3 resource_mapping_tool.py
    ```
+## Tool Arguments
+* `--binaries-path` **[Optional]** Path to QT Binaries necessary for PySide,
```

Comment:
Note: Need to:

a) make a docs change for this
b) call this out as a breaking change (as you are changing existing parameter names).

Reply:
we don't have this in public doc, so it is not a breaking change, but will make docs change to add it.